source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/__init__.py

```diff
@@ -49,3 +54,23 @@
         "rl_games_cfg_entry_point": f"{agents.__name__}:rl_games_camera_ppo_cfg.yaml",
     },
 )
+
+gym.register(
+    id="Isaac-Cartpole-ResNet18-Camera-v0",
+    entry_point="omni.isaac.lab.envs:ManagerBasedRLEnv",
+    disable_env_checker=True,
+    kwargs={
+        "env_cfg_entry_point": CartpoleResNet18CameraEnv,
```

Comment:
Please make changes accordingly everywhere. These aren't env but their configs :)

```suggestion
        "env_cfg_entry_point": CartpoleResNet18CameraEnvCfg,
```